models/hr/src/index.ts

```diff
@@ -149,8 +158,15 @@ export class TRequest extends TAttachedDoc implements Request {
     tzDueDate!: TzDate
 }
 
+@Model(hr.class.PublicHoliday, core.class.Doc, DOMAIN_HR)
+export class TPublicHoliday extends TDoc implements PublicHoliday {
```

Comment:
I think we must have value here with values 1/-1, for example in case of situations then weekend is really marked as working day for some period. Let's make it in next PR.

Reply:
So we just have an option to mark weekend as working day?
I can implement similar popup for working days to be marked only on weekends.